src/app/app-routing.module.ts

```diff
@@ -11,6 +11,10 @@ const routes: Routes = [
     pathMatch: 'full'
   },
   {
+    path: 'list',
```

Comment:
This is just for testing the lazy loading of list page. You can ignore this. Not effecting anything.